src/lib/frontend/d_cnf.ml

```diff
@@ -1337,7 +1337,7 @@ let rec mk_expr
             end
 
           | B.Distinct, _ ->
-            E.mk_distinct ~iff:true (List.map (fun t -> aux_mk_expr t) args)
+            E.mk_distinct ~iff:true (List.map aux_mk_expr args)
```

Comment:
Same as above, can we undo this? There shouldn't be this sort of unrelated cleanup in a hotfix.